LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java

```diff
@@ -128,12 +128,13 @@ public void onDraw(UI ui, PGraphics pg) {
       if (this.panelLabelsVisible.getValueb()) {
         pg.pushMatrix();
         LXVector centroid = p.centroid;
-        pg.translate(centroid.x, centroid.y, centroid.z);
+        pg.translate(centroid.x * 1.12f, centroid.y, centroid.z * 1.02f);
```

Comment:
Can you turn these two constants into variables with units in the name?

Reply:
Comments added, thanks.